examples/minimum-viable-router/src/main.rs

```diff
@@ -0,0 +1,85 @@
+use route_rs_packets::EthernetFrame;
+use route_rs_runtime::link::primitive::{InputChannelLink, OutputChannelLink};
+use route_rs_runtime::link::{LinkBuilder, PacketStream, TokioRunnable};
+use tokio::task::JoinHandle;
+
+mod route;
+
+fn main() {
+    let mut runtime = tokio::runtime::Builder::new()
+        .threaded_scheduler()
+        .enable_all()
+        .build()
+        .unwrap();
+
+    let ingress_wan = crossbeam::channel::unbounded();
+    let ingress_lan = crossbeam::channel::unbounded();
+    let ingress_host = crossbeam::channel::unbounded();
+
+    let egress_wan = crossbeam::channel::unbounded();
+    let egress_lan = crossbeam::channel::unbounded();
+    let egress_host = crossbeam::channel::unbounded();
+
+    drop(ingress_wan.0);
+    drop(ingress_lan.0);
+    drop(ingress_host.0);
+
+    run_route(
+        vec![ingress_wan.1, ingress_lan.1, ingress_host.1],
+        vec![egress_wan.0, egress_lan.0, egress_host.0],
+        &mut runtime,
+        route::Route::new(),
+    );
+
+    assert_eq!(egress_wan.1.len(), 0, "Egress WAN is not empty");
+    assert_eq!(egress_lan.1.len(), 0, "Egress LAN is not empty");
+    assert_eq!(egress_host.1.len(), 0, "Egress Host is not empty");
+
+    println!("Congratulations! Your router successfully does nothing!")
+}
+
+// TODO: Move this to route-rs-runtime
```

Comment:
This looks exactly like what already exists in the runtime already under the utils called "runner"

Reply:
`runner` doesn't provide a way to configure the inputs to the overall `Link`. I suppose you could put it all into the thunk that returns the `Link`, but I'm trying to keep that logic inside the library.

Once we have `af_packet`, this method will handle setting up all of those links and take the socket paths instead of channels.

Reply:
May be able to refactor this thing to call `runner` though.

Reply:
Hmm, I just want runner to be the thing that we use.  I'm not married to the current implementation of runner per se, just that this commit stack should make runner fit our purpose here. Let me think on it for a few days and find a way to merge everything. This use case is valid.